packages/terra-form-textarea/src/TextareaField.jsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import IconError from 'terra-icon/lib/icon/IconError';
 import Field from 'terra-form-field';
-
+import IconError from 'terra-icon/lib/icon/IconError';
```

Comment:
We can revert this, the import order here doesn't matter much. Then we don't need to update the changelog for updating this import order.